src/workflow.ts

```diff
@@ -49,6 +49,15 @@ export interface PgTransactionId {
   txid: string;
 }
 
+interface RecordedOutput extends transaction_outputs {
```

Comment:
I think this can be inlined in the one place it's used

Reply:
Done!